Z80Bench.cs

```diff
@@ -156,5 +156,13 @@ private static void OnMemoryAccess(object sender, MemoryAccessEventArgs e)
                     return;
             }
         }
+
+        public static Func<Z80Processor> GetProcessor = () => {
```

Comment:
These are the 'seams' I was referring to. Would have use virtual/override but with the main method being static that's not really an option, so instead I've attached these public Func<>s. Could tighten up the visibility to protected by having the test classes define their own Z80Bench derivative class to test against